test/test_utils/__init__.py

```diff
@@ -59,9 +59,9 @@ def get_ami_id_ssm(region_name, parameter_path):
 PT_GPU_PY3_BENCHMARK_IMAGENET_AMI_US_WEST_2 = "ami-02d9a47bc61a31d43"
 # Since latest driver is not in public DLAMI yet, using a custom one
 NEURON_UBUNTU_18_BASE_DLAMI_US_WEST_2 = get_ami_id_boto3(region_name="us-west-2", ami_name_pattern="Deep Learning Base AMI (Ubuntu 18.04) Version ??.?")
-# Habana Base v1.3 ami
-UBUNTU_18_HPU_DLAMI_US_WEST_2 = "ami-08e564663ef2e761c"
-UBUNTU_18_HPU_DLAMI_US_EAST_1 = "ami-06a0a1e2c90bfc1c8"
+# Habana Base v1.5 ami
```

Comment:
We will make it backward compatible in the future. Can you please add a comment that mentions that AMI ids for all the other SynAI versions?

Reply:
done

Reply:
Thanks a lot, this is extremely beneficial.